src/visitors/uglifyPure.js

```diff
@@ -1,16 +1,18 @@
 import annotateAsPure from "@babel/helper-annotate-as-pure"
 
 import { useUglifyPure } from '../utils/options'
-import { isStyled, isHelper } from '../utils/detectors'
+import { isStyled, isHelper, isInjectGlobalHelper } from '../utils/detectors'
 
 export default (path, state) => {
     if(useUglifyPure(state)){
         if(isStyled(path.node,state) || 
           isStyled(path.node.callee,state) || 
-          isHelper(path.node.callee,state) 
+          isHelper(path.node.callee,state) ||
+          isInjectGlobalHelper(path.node.callee,state) 
```

Comment:
i dont think we should annotate `injectGlobal`, its clearly effectful and removing calls to it will break most apps

Reply:
I was thinking the same. @philpl why was you thinking we should annotate injectGlobal?

Reply:
Yeah let's not annotate `injectGlobal` it is not pure 😅

Reply:
True! My mistake :) let’s go ahead with this as it supports every other helper now